docs/_docs/Unofficial Sidecar Academy/Sidecar-Academy.md

```diff
@@ -0,0 +1,106 @@
+
+
+## What is the same and what is different
+
+While the Sidecar Academy will share the same materials as LDSSA, there are a few important differences as follows:
+
+
+1. There will be a different community. Many of the instructors that are available on the LDSSA will not be directly available in the Sidecar Academy.
+
+2. The students will need to help each other a LOT. Since there will be less instructors available, this is a necessity.
+
+3. It will be completely free and remote.
+
+4. The feedback is provided through the material's self-grading system so students will need to keep track of their progress autonomously
+
+5. There will not be bootcamp nor final capstone project
+
+6. Hackathons will follow a format similar to Kaggle-style challenges (More details in section below)
+
+
+## Introduction session
+The introduction session is an online session that kicks off the Sidecar academy, which will take place on the 24th of October at 3pm. In this session all rules and schedule present in this page will be presented, together with any practical details for the current edition.
+
+Additionally we will go over and answer any questions related to the following topics:
+- Course' structure
+- Code of Conduct
+- How to ask for and give help
+- Hackathons
+
+Please keep in mind that this session is not compulsory. Nonetheless, we encourage you to participate and get to know the staff behind the course and the fellow students who will be with you in this 8-month journey.
+
+
+## Hackathons 
+
+Each Hackathon will take place during three days. A few days prior to the starting date a form will have to be filled to confirm the interest in participating in each Hackathon. Teams will be randomly assigned based on the number of participants.
+
+Once you are assigned with a team, it is up to you and your fellow teammates to organise yourself into working together and solving the problems through joint effort. Teams should submit their results to a Kaggle competition that will be created by the LDSA team. A classification table will also be available to compare the various approaches to the problem. Every team is encouraged to post their solution notebook at the end of each Hackathon.
+
+
+## Schedule
+
+The master schedule google calendar is [here](https://calendar.google.com/calendar/embed?src=j8t3fmuv53pppvf2vcnfji4tl8%40group.calendar.google.com&ctz=Europe%2FLisbon)!
+You can add this calendar to most calendar apps so that you can be sure that you stay aware of all the important dates.
+
+## Late policy
+If something happens and you could not follow the deadline, you can still ask questions about the previous week's SLU. Just remember to post it in the right Slack channel!
+
+
+# Frequently Asked Questions (FAQ)
+
+##### 1. I just want to check that the whole course is 100% remote right?
+Everything is 100% remote and online!
+
+##### 2. If something happens and I could not follow the deadline, would that be okay?
```

Comment:
This repeats what is mentioned in the Late Policy. Just leaving a comment here in case it was not intended. If it is, ignore this comment.